components/openzwave-adapter/src/lib.rs

```diff
@@ -352,6 +385,12 @@ impl OpenzwaveAdapter {
                         if kind.is_none() { continue }
                         let kind = kind.unwrap();
 
+                        let mut tags = HashSet::new();
+                        let label = vid.get_label();
+                        if !label.is_empty() {
+                            tags.insert(TaxoId::<TagId>::new(&vid.get_label()));
```

Comment:
also, wondering if this should be a tag or a property with a `label` key (or something else that we agree with our frontend friends). @azasypkin @gmarty ?